src/cms-content/learn/index.ts

```diff
@@ -161,6 +162,22 @@ export function getMoreStudies(caseStudyId: string): CaseStudy[] {
 
 export const caseStudiesContent = caseStudies as CaseStudiesContent;
 
+/* We have previously set certain case studies as hidden via hard-coding a filter.
+  We now have the ability to hide them via a switch in the CMS.
+  This function+filter make sure that we dont render a table of contents item
+  for categories in which all case studies are hidden:
+*/
+function showCategory(category: CaseStudyCategory) {
+  return (
+    category.caseStudies.filter((study: CaseStudy) => study.showCaseStudy)
+      .length > 0
+  );
+}
+
+export const categoriesWithStudies = caseStudiesContent.categories.filter(
```

Comment:
@pnavarrc re: your comment below-- see here--the categories are already exported with the show/hide logic